test/test.py

```diff
@@ -0,0 +1,5 @@
+
+# added comment
+
+def test():
```

Comment:
make it rhyme

Reply:
i really can't make things rhyme... sorry